document/web-api/index.bs

```diff
@@ -125,6 +135,36 @@ Note: This algorithm accepts a {{Response}} object, or a
      1. Return |returnValue|.
 </div>
 
+<h2 id="serialization">Serialization</h2>
+
+On the Web, {{Module}} has the <code>[<a extended-attribute>Serializable</a>]</code> extended attribute.
```

Comment:
Can't this be in its IDL? This seems somewhat unclear.

Reply:
Well, the JS API specification is trying to avoid referencing anything which is Web-specific. [Serializable] is defined in HTML. I'm not sure how to put this in its IDL without violating that layering.

Reply:
If these objects are not defined in IDL then you need to patch HTML directly as you did with BigNum.

Reply:
(I said as much on the original PR.)

Reply:
They are defined in IDL.

Reply:
Here's the IDL: https://webassembly.github.io/spec/js-api/index.html#module

Reply:
It also has Exposed=(Window,Worker,Worklet) so I think you're failing somewhat?

Reply:
@annevk Well, there's a plan for that: https://github.com/heycam/webidl/issues/468 . This layering is still a goal.

Reply:
Okay. I think you need to make this something like "Web user agents must augment the Module interface with the [Serializable] extended attribute." so it's an actual requirement and not a statement of fact that isn't based on any established facts...

Reply:
Thanks for the wording; I've switched to use it.